src/main/java/gregtech/loaders/recipe/VanillaOverrideRecipes.java

```diff
@@ -696,6 +757,28 @@ private static void removeCompressionRecipes() {
             ModHandler.removeRecipeByName("minecraft:smooth_red_sandstone");
             ModHandler.removeRecipeByName("minecraft:bookshelf");
             ModHandler.removeRecipeByName("minecraft:pillar_quartz_block");
+            ModHandler.removeRecipeByName("minecraft:sea_lantern");
+            ModHandler.removeRecipeByName("minecraft:chiseled_sandstone");
+
+            // Slab replacement
```

Comment:
This slab replacement is not doing the wood slabs. Is that enabled differently, and if so, so we consolidate them?

Reply:
Wood slabs are handled in a separate handler dedicated entirely to wood types and handling all wood-related items and recipes in one place